components-api/src/main/scala/pl/touk/nussknacker/engine/api/component/AdditionalComponentsUIConfigProvider.scala

```diff
@@ -0,0 +1,7 @@
+package pl.touk.nussknacker.engine.api.component
+
+trait AdditionalComponentsUIConfigProvider {
+
+  def getAllForCategory(category: String): Map[String, SingleComponentConfig]
```

Comment:
We need the:
```
getAll: Map[ComponentId, SingleComponentConfigWithoutComponentId]
```
, otherwise how to deal with components deduplicated across multiple processing types?